### ISSUE TITLE
L1: transfer transaction hangs console

### ISSUE THREAD

**Issue by nerveharp** [urbit/bridge#965]
- Transferred a star on L1 targeting a metamask wallet
- Logged in via MM and accepted the point
- See a near-infinite log of errors; console crashes
- The [transaction completes](https://ropsten.etherscan.io/tx/0xa10a7078aaad433150decc98a062981e0c3daf64c8657338068d1401fafcf952) but the front-end never responds.

Visible on Ropsten; Chrome 98, MacOS.

![image](https://user-images.githubusercontent.com/748181/153744477-f7d81ff9-722c-4bd3-84c2-2ac5ceaf4d81.png)

```
HTTPTransport.js:91          POST https://roller-tmp.urbit.org/v1/roller net::ERR_INSUFFICIENT_RESOURCES
(anonymous) @ HTTPTransport.js:91
(anonymous) @ HTTPTransport.js:46
(anonymous) @ HTTPTransport.js:27
(anonymous) @ HTTPTransport.js:21
i @ HTTPTransport.js:17
t.sendData @ HTTPTransport.js:80
(anonymous) @ RequestManager.js:96
(anonymous) @ RequestManager.js:33
(anonymous) @ RequestManager.js:14
(anonymous) @ RequestManager.js:8
r @ RequestManager.js:4
e.request @ RequestManager.js:83
(anonymous) @ Client.js:109
(anonymous) @ Client.js:33
(anonymous) @ Client.js:14
s @ Client.js:5
Promise.then (async)
l @ Client.js:7
(anonymous) @ Client.js:8
r @ Client.js:4
e.request @ Client.js:100
value @ index.ts:489
getSpawningFor @ index.ts:786
(anonymous) @ useRoller.ts:609
c @ runtime.js:63
(anonymous) @ runtime.js:294
(anonymous) @ runtime.js:119
r @ asyncToGenerator.js:3
o @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ useRoller.ts:597
(anonymous) @ useControlledPointsStore.js:52
c @ runtime.js:63
(anonymous) @ runtime.js:294
(anonymous) @ runtime.js:119
r @ asyncToGenerator.js:3
o @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ AcceptTransfer.tsx:55
(anonymous) @ useEthereumTransaction.js:308
c @ runtime.js:63
(anonymous) @ runtime.js:294
(anonymous) @ runtime.js:119
r @ asyncToGenerator.js:3
o @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
(anonymous) @ useEthereumTransaction.js:305
Rl @ react-dom.production.min.js:262
t.unstable_runWithPriority @ scheduler.production.min.js:18
Gi @ react-dom.production.min.js:122
Pl @ react-dom.production.min.js:261
(anonymous) @ react-dom.production.min.js:261
N @ scheduler.production.min.js:16
_.port1.onmessage @ scheduler.production.min.js:12
Show 26 more frames

VM584 react_devtools_backend.js:4061 failed to fetch controlled points Error: Failed to fetch
    at new t (Error.js:24:28)
    at t.<anonymous> (HTTPTransport.js:117:39)
    at HTTPTransport.js:46:23
    at Object.throw (HTTPTransport.js:27:53)
    at o (HTTPTransport.js:19:56)
```

**Comment by nerveharp** [comment]
This eventually "resolved" and gave me the following feedback. Marking as non-release blocking, but annoying.

![image](https://user-images.githubusercontent.com/748181/153744612-38a70155-fd6e-48f4-86b2-d9ac2860d5ce.png)

**Comment by lukebuehler** [comment]
I noticed the same, I saw thousands of failed preflight checks to the roller api.
![bridge](https://user-images.githubusercontent.com/542828/154835107-bcda9494-5777-4c5d-b735-1b6142ae6599.png)

**Comment by pkova** [comment]
Fixed by #1052.
